### PR TITLE
tests: Speed up CSS tests by using smaller Japanese font file

### LINKED ISSUE #11452: ol_japanese_iroha_a.html takes >10s to run to completion
Both the test file and the reference file take >10s each, so the total time for running that reftest regularly exceeds 23s when I run them locally.

```
godot2:servo-pristine jdm$ time ./mach run tests/wpt/mozilla/tests/css/ol_japanese_iroha_a.html -x
Shutting down the Constellation after generating an output file or exit flag specified

real    0m13.147s
user    0m27.253s
sys 0m2.890s
godot2:servo-pristine jdm$ time ./mach run tests/wpt/mozilla/tests/css/ol_japanese_iroha_ref.html -x
Shutting down the Constellation after generating an output file or exit flag specified

real    0m10.704s
user    0m22.859s
sys 0m2.489s
```

### PR DESCRIPTION
**Pull request by talklittle**
Replace the Japanese font with a much smaller font file to speed up relevant CSS tests.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes fix #11452 (github issue number if applicable).

- [ ] There are tests for these changes OR
- [X] These changes do not require tests because modifying test to swap font

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/11630)

### CODE REVIEW

File: tests/wpt/mozilla/tests/css/ol_japanese_iroha_ref.html [hunk 1]
```diff
@@ -20,10 +18,10 @@
 </head>
 <body>
 <ol>
-    <li>&#x3044;. Gryffindor</li>
```
Comment (SimonSapin) [inline tests/wpt/mozilla/tests/css/ol_japanese_iroha_ref.html]: Why this change? I *think* that’s what’s making `ol_japanese_iroha_a.html` fail.

Comment (talklittle) [inline tests/wpt/mozilla/tests/css/ol_japanese_iroha_ref.html]: I changed it because the new font was causing the reftest to fail. After this change it no longer fails. I'm guessing the presence of the `.` is font- and browser-specific. In Firefox it shows a "Japanese comma" instead of a period.

Servo with new font:
screenshot from 2016-06-08 10-06-08

Servo with old font:
screenshot from 2016-06-08 10-10-20


Firefox with new font:
screenshot from 2016-06-08 09-36-22

Firefox with old font:
firefox-old-font

**Comment by talklittle** [comment]
I guess the new font is triggering `RenderingMode::Plain` instead of `RenderingMode::Suffix` for some reason. https://github.com/servo/servo/blob/master/components/layout/generated_content.rs

**Comment by jdm** [comment]
@SimonSapin This has been waiting in your queue for a long time now.

**Comment by SimonSapin** [comment]
Sorry, I got this PR confused with #8374 which was blocked because of test failures.

> I guess the new font is triggering RenderingMode::Plain instead of RenderingMode::Suffix for some reason. https://github.com/servo/servo/blob/master/components/layout/generated_content.rs

There is no reason for that to happen. And I’ve verified in a debugger that the code with `RenderingMode::Suffix(".\u{00a0}")` is indeed executed. It looks like using this font exposes a layout/rendering bug.

I’m somewhat opposed to blindly changing a reference file to match a rendering bug in a test case.

This is what the test with this PR looks for me, with `font-size: 4em` added:

screenshot from 2016-08-16 17-14-28

I don’t know why the latin letters are light grey instead of black. There is no declaration for the `color` CSS property.

During the page load, for a about one second, the rendering looks like this (if I’m quick enough with the screenshot button):

screenshot from 2016-08-16 17-13-50

Maybe it while the font isn’t loaded yet? The dot is present during that time.

@pcwalton any idea what’s going on?

**Comment by pcwalton** [comment]
@SimonSapin Probably the font hasn't loaded yet. This would be an excellent use case for the thread profiling tool I'm working on :)

**Comment by SimonSapin** [comment]
Yes, my second screenshot is when the font hasn’t loaded yet. (This is taken manually, it’s not what the test harness gets.) The thing is why, when the font does eventually load (my first screenshot), do the dot on space disappear?

**Comment by pcwalton** [comment]
Oh, that's weird. Does it reproduce with incremental layout off?

**Comment by SimonSapin** [comment]
With `-i`, yes.

**Comment by jdm** [comment]
Tracking the bug this exposed in #15273. Closing because this doesn't appear to be going anywhere.
